packages/zowe-explorer/src/utils/LoggerUtils.ts

```diff
@@ -72,6 +72,11 @@ export class ZoweLogger {
         await this.zeOutputChannel.dispose();
     }
 
+    public static async getLogSetting(): Promise<string> {
+        this.zeLogLevel = await vscode.workspace.getConfiguration().get("zowe.logger");
```

Comment:
Sorry I missed this when I reviewed earlier - should we use `SettingsConfig.getDirectValue` here to access the `zowe.logger` setting? I think we've been trying to use that method instead of `vscode.workspace.getConfiguration()` which loads the entire configuration and was problematic in some Theia versions.

Reply:
@t1m0thyj I tried using that and VSC crashes on initialization.  could be that `SettingsConfig.getDirectValue` uses vsc getConfiguration but at a zowe level and since not fully activated yet that level hasn't been completely built.

Reply:
Merging now, will investigate `SettingsConfig.getDirectValue` on the other PR 🙂